packages/wonder-blocks-dropdown/components/dropdown-core.js

```diff
@@ -595,34 +642,17 @@ export default class DropdownCore extends React.Component<
                 ]}
             >
                 {this.maybeRenderSearchTextInput()}
-                {items.map((item, index) => {
-                    if (SeparatorItem.isClassOf(item.component)) {
-                        return item.component;
-                    } else {
-                        if (item.focusable) {
-                            focusCounter += 1;
-                        }
-                        const focusIndex = focusCounter - 1;
-                        return React.cloneElement(item.component, {
-                            ...item.populatedProps,
-                            key: index,
-                            ref:
-                                item.focusable &&
-                                this.state.itemRefs[focusIndex].ref,
-                            role: itemRole,
-                            onClick: () => {
-                                this.handleClickFocus(focusIndex);
-                                if (item.component.props.onClick) {
-                                    //$FlowFixMe
-                                    item.component.props.onClick();
-                                }
-                                if (item.populatedProps.onClick) {
-                                    item.populatedProps.onClick();
-                                }
-                            },
-                        });
-                    }
-                })}
+
+                {itemsList.length < 100 ? (
+                    this.renderAllItems(itemsList)
+                ) : (
+                    <DropdownCoreVirtualized
```

Comment:
How bad would it be to just always have it be virtualized? I'm really not excited about having two code paths for everything and having the potential for stuff to break in strange ways as we can't anticipate the change. If we do want this separated what can we do to ensure that we're able to keep these two modes in sync?

Reply:
Yeah, that's an excellent point. I'm going to change that to always use virtualized lists. At the end, if there are a few items, then the library should be able to detect when to virtualize items.